src/style/footer/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import './style.css';
+import logo from './image/LOGO.svg'
+import { FooterBlock } from "./footer";
+import { Separator } from "../separator/separator";
+
+export const Footer = () => (
+
+  <footer>
+    <div  className="footer-container">
+      <div className="logo-container">
+        <img src={logo} alt="logo" />
+      </div>
+      <div className="itens-container">
+      <div>
+        <FooterBlock
+          title="Comprar"
+          items={['Casas', 'Apartamentos', 'Lançamentos', 'Melhores negócios']}
+        />
+        <Separator size="large" />
+        <Separator size="large" />
+        <Separator size="large" />
+        <div>
+          <FooterBlock
+            title="Alugar"
+            items={['Casas', 'Apartamentos', 'Lançamentos', 'Melhores negócios']}
+          />
+        </div>
+      </div>
```

Comment:
nao precisa dessas divs ne?

```suggestion
        <FooterBlock
          title="Comprar"
          items={['Casas', 'Apartamentos', 'Lançamentos', 'Melhores negócios']}
        />
        <Separator size="large" />
        <Separator size="large" />
        <Separator size="large" />
          <FooterBlock
            title="Alugar"
            items={['Casas', 'Apartamentos', 'Lançamentos', 'Melhores negócios']}
          />
```